lib/cldr/export/data/country_codes.rb

```diff
@@ -0,0 +1,28 @@
+module Cldr
+  module Export
+    module Data
+      class CountryCodes < Base
+        def initialize
+          super(nil)
+          update(:CountryCodes => country_codes)
+        end
+
+        private
+
+        def country_codes 
+          doc.xpath("//codeMappings/*").each_with_object({}) do |node, hash|
+            if (node.name == "territoryCodes")
```

Comment:
```suggestion
            if node.name == "territoryCodes"
```

No need for those in ruby ;)

Reply:
Can do!